src/status_im/ui/components/list_item/views.cljs

```diff
@@ -1,81 +1,378 @@
+
+                (vector? accessory)
+                accessory
+
+                :else nil)])
+           {:key accessory}))))))
+
+;; every key is optional - use as needed
+;; combination of around 4 related keys are enough for most cases
+
+;; type
+;; :default (default), :small, or :section-header
+;; - :section-header
+;;   specifying only these is sufficient
+;;   {:title "Section title" :type :section-header}
+;;   optionally set `container-margin-top/bottom`
+
+;; theme
+;; :default(default), :blue, :action, :action-destructive
+
+;; container-margin-top
+;; container-margin-bottom
+;; number - 0 by default
+;; usually the first item has top margin
+;; the last item has bottom margin
+
+;; icon
+;; any one of keyword representing :main-icon/icon, photo-path, component
+;; if component make sure to make it 40x40 size
+
+;; title-prefix
+;; any one of keyword representing an vector-icon/icon,
+;; :main-icons/tiny-icons(16x16) are preferred(when it has 4px :margin-top)
+;; any other vector-icon/icon is accepted though(when it is better to
+;; specify height(best to keep it <= 20) see related height/width below
+;; string, keyword(gets converted to string),
+;; number(gets converted to string), or component
+
+;; title-prefix-width
+;; title-prefix-height
+;; optional width/height for when title/prefix is not a tiny-icon
+;; i.e. when icon height/height > 16, or when component
+;; do not specify if title-prefix is tiny-icon
+
+;; title
+;; any one of string, keyword representing translated string in the form of
+;; :t/{translation-key-in-translation-files},
+;; keyword(gets converted to string),
+;; number(gets converted to string), or
+;; component - when component is used best to keep the style similar
+;; to `styles/title-row-container` and/or `styles/title`
+
+;; title-color-override
+;; colors/color - only occasionally needed, self-explanatory
+
+;; title-row-accessory
+;; component - especially made for chat list item, but may serve other
+;; purpose in the unlikely future. Wrapper already has 2px :margin-top
+;; best to keep it <= 18px high
+
+;; subtitle
+;; any one of string, keyword representing translated string in the form of
+;; :t/{translation-key-in-translation-files},
+;; keyword(gets converted to string),
+;; number(gets converted to string), or
+;; component - when component is used best to keep the style similar
+;; to `styles/subtitle-title-row-container` and/or `styles/subtitle`
+
+;; subtitle-max-lines
+;; integer - 1 by default - self-explanatory
+
+;; subtitle-row-accessory
+;; component
+;; made specially for chat-list to hold unread messages counter
+
+;; content
+;; component - to replace entire title-column area
+;; TODO - consider removing, as it is redundant now that
+;; title/subtitle elements can be component as well
+;; just make sure to keep in mind the note made on
+;; component case on those keys above
+
+;; accessories
+;; vector of :chevron, :check, :more, string, number, keword or component
```

Comment:
keyword

Reply:
thanks for catching the typo.. will fix it

Reply:
done